### ISSUE TITLE
Can not simulate the queue congestion of bottleneck section

### ISSUE THREAD

**Issue by Shiyunyang-seu** [eclipse/sumo#7212]
Hi, when the number of lanes in some sections of the expressway decreases, there should be queues in the bottleneck sections, but although I continue to increase the input traffic (the input flow also contains some trucks), there also be no queues (this is not consistent with the reality). 
I think the possible reason is that the speed of the vehicles in the junction section is too fast(Or vehicles are easy to go through the junction), so I use a short-length edge instead of the junction section, but there is still no queue. How can I solve this problem?
Thanks in advance.
![no queue](https://user-images.githubusercontent.com/55737075/85222446-9ebc8f00-b3ed-11ea-80f0-e49832642ae7.png)

**Comment by namdre** [comment]
your screenshot does not show a lane reduction. The southbound flow that is visible in your image continuously uses 2 lanes.

**Comment by Shiyunyang-seu** [comment]
Hello, the above figure is just one of the cases. I tested a lot of cases at the same time (there was no queue when the number of upstream lanes was reduced)
[queue.zip](https://github.com/eclipse/sumo/files/4809405/queue.zip)

**Comment by Shiyunyang-seu** [comment]
Hello, I found a problem. I set up an E2 detector on the road, and defined vehicles passing through the road section through flow. I continuously increased the flow value (2000,5000,10000...) and other ODs(also passed through the road section). However, I found that the vehicles entering the road are almost the same. Does it mean that the maximum number of vehicles can only be input about 140 every 300 seconds in one road.
![flow](https://user-images.githubusercontent.com/55737075/85230412-78651680-b422-11ea-8bfb-2ec0f60da0c6.png)
![e2](https://user-images.githubusercontent.com/55737075/85230414-7dc26100-b422-11ea-88b6-76a92889c0a8.png)
